bookkeeper-server/src/main/resources/LICENSE.bin.txt

```diff
@@ -1,302 +0,0 @@
-
```

Comment:
any reason why removing this one and NOTICE.bin.txt?

Reply:
This seems a dup to README and NOTICE under root dir. so use the ones under root dir.  If this is not right, will add them back.